src/OpenTelemetry/Metrics/ExportModeAttribute.cs

```diff
@@ -0,0 +1,32 @@
+// <copyright file="ExportModeAttribute.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+
+namespace OpenTelemetry.Metrics
+{
+    public sealed class ExportModeAttribute : Attribute
```

Comment:
We should probably specify `[AttributeUsage(...)]` on the Attributes we have defined.

Reply:
I didn't know that until now...
Will take a look, thanks!

Reply:
Covered by #2357.